functions/go/set-image/go.mod

```diff
@@ -0,0 +1,12 @@
+module github.com/GoogleContainerTools/kpt-functions-catalog/functions/go/set-namespace
+
+go 1.16
+
+require (
+	github.com/stretchr/testify v1.7.0
+	sigs.k8s.io/kustomize/api v0.10.1
+	sigs.k8s.io/kustomize/kyaml v0.13.0
+
+)
+
+replace sigs.k8s.io/kustomize/kyaml v0.13.0 => github.com/mengqiy/kustomize/kyaml v0.15.0
```

Comment:
I have move the fork. You can change it to https://github.com/GoogleContainerTools/kpt-functions-catalog/blob/master/functions/go/export-terraform/go.mod#L6

Reply:
Done